.circleci/docker/ubuntu-cuda/Dockerfile

```diff
@@ -93,5 +93,9 @@ ENV TORCH_NVCC_FLAGS "-Xfatbin -compress-all"
 # Install LLVM dev version (Defined in the pytorch/builder github repository)
 COPY --from=pytorch/llvm:9.0.1 /opt/llvm /opt/llvm
 
+ADD ./common/install_openssl.sh install_openssl.sh
+ENV OPENSSL_ROOT_DIR /opt/openssl
```

Comment:
This variable should make it so that cmake should opt to use `/opt/openssl` as the openssl library to build with